requirements.txt

```diff
@@ -10,7 +10,7 @@ OctoBot-Tentacles-Manager>=2.7.0, <2.8
 trading-backend>=1.0.12
 
 # Exchange connection requirements
-ccxt==1.95.36 # always ensure real exchanges tests (in tests_additional) are passing before changing the ccxt version
+ccxt==2.1.92 # always ensure real exchanges tests (in tests_additional) are passing before changing the ccxt version
```

Comment:
if we can, I think it's better not to update the ccxt version at the same time to be sure that the code works as expected with the current ccxt version and then update ccxt (not to confuse ccxt and code issues)

Reply:
Okay we can change that later as it works with both versions